unstructured_api_tools/pipelines/templates/pipeline_api.txt

```diff
@@ -29,11 +40,33 @@ async def pipeline_1(request: Request, file: UploadFile = File(),
 {{param}}: List[str] = Form(default=[]),
 {% endfor %}
 ):
-
+    
+    media_type = request.headers.get("accept")
+    {% if response_type %}
+    default_response_type = "{{response_type}}"
+    media_type = default_response_type if media_type == "*/*" else media_type
+    {% endif %}
     text = file.file.read().decode("utf-8")
+
+    {% if response_type %}
+    response = pipeline_api(text, {% for param in multi_string_param_names %}m_{{param}}={{param}}, {% endfor %} response_type=media_type,)
+    {% else %}
     response = pipeline_api(text, {% for param in multi_string_param_names %}{{param}}, {% endfor %})
+    {% endif %}
 
-    return response
+    if _is_confict_response_type(media_type, type(response)):
+        return PlainTextResponse(
+            content=f"Conflict in media type {media_type} with response type {type(response)}.\n",
+            status_code=status.HTTP_409_CONFLICT,
+        )
+    valid_response_types = ["application/json", "text/csv", "*/*"]
+    if media_type in valid_response_types:
+        return response
+    else:
+        return PlainTextResponse(
+            content=f"Unsupported media type {media_type}.\n",
+            status_code=status.HTTP_415_UNSUPPORTED_MEDIA_TYPE,
+        )
```

Comment:
All of the above (from line 57) can be in a `{% if response_type %}` block,
else we can just

    return response

as is currently the case.